schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+   id INT PRIMARY KEY,
+   name           TEXT,
+   date_of_birth   DATE,
+   escape_attempts INT,
+   neutered         BOOLEAN,
+   weight_kg       FLOAT
```

Comment:
- [ ] Great job so far 👍 Kindly use `decimal` for your `weight_kg` as given in the instructions 👍